src/main/kotlin/com/depromeet/makers/domain/use_case/UseCase.kt

```diff
@@ -0,0 +1,5 @@
+package com.depromeet.makers.domain.use_case
+
+interface UseCase<I, O> {
```

Comment:
```suggestion
fun interface UseCase<I, O> {
```

함수형으로 작성해도 좋을 것 같아요..!

Reply:
오 좋습니다..!! 변경해두겠습니다